homeassistant/components/ecobee/weather.py

```diff
@@ -49,6 +53,11 @@ async def async_setup_entry(
 class EcobeeWeather(WeatherEntity):
     """Representation of Ecobee weather data."""
 
+    _attr_native_pressure_unit = PRESSURE_HPA
+    _attr_native_temperature_unit = TEMP_FAHRENHEIT
+    _attr_native_visibility_unit = LENGTH_METERS
+    _attr_native_wind_speed_unit = SPEED_MILES_PER_HOUR
```

Comment:
I'm not really sure about these, the units are not documented: https://www.ecobee.com/home/developer/api/documentation/v1/objects/WeatherForecast.shtml